app/js/components/manageApps/Addon.jsx

```diff
@@ -83,6 +90,71 @@ class Addon extends Component {
 
   }
 
+  fetchCurrentUser(){
+    const apiHelper = new ApiHelper(null);
+    const getUserData = new Promise(function(resolve, reject) {
+      apiHelper.get('/v1/session').then(response => {
+        resolve(response);
+      });
+    });
+    return getUserData;
+  }
+
+  checkIsAdmin() {
+    this.fetchCurrentUser().then((response) => {
+      if (response.user) {
+        const user = response.user;
+        let userPrivileges = [];
+        let userCanManageModules;
+        let userHasPermittedRoles;
+        userPrivileges = userPrivileges.concat(user.privileges);
+        if (user.roles.length > 0){
+          const roles = user.roles;
+          const userRoles = roles.find((userRole) => {
+            if (userRole.display === "System Developer" || userRole.display === "Provider" ) {
+              return userHasPermittedRoles = true;
+            }
+            return userHasPermittedRoles = false;
+          });
+        }
+        if (userPrivileges.length > 0) {
```

Comment:
`if(userPrivileges.length > 0)` is equivalent to `if(userPrivileges.length)`

Boolean values are some form of binary integer values so length 0 evaluates to false

Reply:
if I do this I will be evaluating if the length not if the array has content,
doing `userPrivileges.length` will return an integer which is not a good way of checking conditions but doing `userPrivileges.length > 0` will be a boolean checking if the threshold of the length required to proceed is true or false